docs/en/patterns/lists.md

```diff
@@ -75,8 +75,20 @@ tutorial or instructions — you can use the class ```.p-list-step```.
     View example of the stepped list pattern
 </a>
 
+## Stepped list: detailed
+
+Stepped list should be used for step by step instructions. This pattern is best
+used on a grey strip as the description sections are displayed in a white box.
```

Comment:
I think this should say 'light strip' as that is the name of the class to use? This confused me doing canonical recently as it was mentioned I should use the grey strip class and I was wondering why `.p-strip--grey` wasn't working.